lib/LTO/ThinLTOCodeGenerator.cpp

```diff
@@ -239,6 +244,49 @@ static void optimizeModule(Module &TheModule, TargetMachine &TM,
   PM.run(TheModule);
 }
 
+static void optimizeModule2(Module &TheModule, TargetMachine &TM,
```

Comment:
should use a more descriptive function name

Reply:
This function does have a more descriptive name in https://github.com/RadeonOpenCompute/llvm/pull/129, because I was cherry picking, I only have the initial change  (first commit) for opt here. So there are some outdated parts in the first commit of this PR, which are revised in PR #129